Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Models/DrugReportStruct.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace HealthcareHospitalManagementSystem.Services
+{
+    public struct DrugReportStruct
+    {
+        public string SafetyReportId { get; set; }
```

Comment:
The task was to have 2 different use cases for class and struct, use class where it is applicable and use struct where it is applicable. In your case, except for duplication of the cases (literally the same entity), there will also be issues with performance, it is recommended not to store strings in struct classes due to the reference nature of the strings. Although the struct is a value type, it will contain references for defined strings and there will be a lot of memory allocated in the heap. So struct does not make sense here.

It is better to use struct for small data structures that can contain int, float, double values, etc, and entities that must come as one onject, examples: Point (int x, int y), in BankAccount example project Transaction (Id, Amount, ...)

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct